Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM registry.ci.openshift.org/ocp/4.9:base
+WORKDIR /go/src/github.com/openshift/aws-efs-utils
+COPY . .
```

Comment:
Is content of the repository actually necessary? IMO nothing needs it.

Reply:
I just need files in `dist` .. will fix it now 👍🏽